modules/scope/variables.tf

```
@@ -0,0 +1,16 @@
+variable "ipam_id" {
+  description = "(Required) The ID of the IPAM for which you're creating this scope."
+  type        = string
+}
+
+variable "description" {
+  description = "(Optional) A description for the scope you're creating."
+  type        = string
+  default     = "My IPAM Scope"
+}
+
+variable "tags" {
+  description = "(Optional) Key-value mapping of resource tags."
+  type        = map(string)
+  default     = { "Name" = "test" }
```

Defaults for tags should just about _always_ simply be an empty map

```suggestion
  default     = {}
```

---

Updated all tags in module variables